### DISCUSSION TITLE
Doc rebuild to Solid vs Astro

### DISCUSSION THREAD

**Discussion by patrickjohnstevens** [nikivdev/go#117] [General]
You mentioned wanting to rebuild using Solid... if you want speed and a template that is already good to go I would recommend Astro. Blazingly fast because it ships zero JS by default. My main thing right now is creating the workflow and habit of learning in public. Astro is the next step up if you want max speed! The Lighthouse scores on it were amazing. I like the documentation on Docusaurous a lot more which is one of the reasons I am starting with it. If I were to build something I would go with Qwik/City over Solid but they are endorsed and created by the same guy so they are both great. Caveat... I'm still learning React so my opinion is not to be taken as expert level. Ha! 

https://github.com/withastro/astro/tree/main/examples/docs

**Comment by nikivdev** [comment]
Oh neat, will take a look into it. Thanks.
